app/cdap/components/WrangleHome/Components/OngoingDataExplorations/utils.ts

```diff
@@ -14,44 +14,93 @@
  * the License.
  */
 
-import { ImportDatasetIcon } from '../WrangleCard/iconStore/ImportDatasetIcon';
-import { IMassagedObject } from './types';
+import DataPrepStore from 'components/DataPrep/store';
+import {
+  IExistingExplorationCardsData,
+  IMassagedObject,
```

Comment:
remove Object keyword from interface, variable and function names...
rename this interface appropriately